### DISCUSSION TITLE
how to use pixijsv5 make 3d anim

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#7174] [Q&A]
how to use pixijsv5 make plane 3d animation

**Comment by xzilja** [comment]
I think you'll need to provide much more info in your question in order to get a good answer on this, but try looking at 3d projection examples like this one https://pixijs.io/examples/#/plugin-projection/cards.js (click on the cards, they will animate). There are more examples there as well. Hope it helps.

**Reply by a6965921** [reply]
> I think you'll need to provide much more info in your question in order to get a good answer on this, but try looking at 3d projection examples like this one https://pixijs.io/examples/#/plugin-projection/cards.js (click on the cards, they will animate). There are more examples there as well. Hope it helps.

i make it in 2d   if use plugin-project     Coordinate size is not control

**Comment by goldenratio** [comment]
worth looking into https://github.com/jnsmalm/pixi3d
